spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -118,3 +118,36 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 
 - Time values in TCK tests are now configurable (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/399[#399]).
 - Transitive dependency on `javax.el-api` has been excluded (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/439[#439]).
+
+[[release_notes_30]]
+== Release Notes for MicroProfile Fault Tolerance 3.0
+
+This release is a major release of Fault Tolerance which includes backward incompatible changes.
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-fault-tolerance/milestone/9[MicroProfile Fault Tolerance 3.0 Milestone]
+
+=== Backward incompatible changes
+
+==== Metric names and scopes changed
+
+The names of the metrics added automatically by MicroProfile Fault Tolerance have been updated to take advantage of support for metric tags which was added to MicroProfile Metrics in version 2.0. As a result, some information which was previously contained in the metric name is now instead included in tags.
+
+In addition, metrics have moved from the `application:` scope to the `base:` scope for consistency with other MicroProfile specifications.
+
```

Comment:
Not sure I want to add this information. In other parts of the spec, we only talk about how fault tolerance adds metrics through the Metrics API, we never discuss or test how MP Metrics exports that information (which indirectly insulates us from any changes they might make to the export format).

Additionally, if we do add it, then, as far as I'm aware, it makes the following changes:
* metrics are no longer exported from `/metrics/application`
* metrics are exported from `/metrics/base`
* in the JSON format, when metrics are retrieved from `/metrics` they appear in the `base` object rather than the `application` object
* in the OpenMetrics format, the names are prefixed with `base_` instead of `application_`

Reply:
Well, I've added it, but I'm not sure if it means we devote too much space to talking about the scope change, when the change of metric names and the addition of tags is the bigger and more important part of the change.

I'm happy to leave it in or take it out again, let me know what you think.